src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -578,7 +578,7 @@ private TypeSymbol BindNamedAttributeArgumentType(AttributeArgumentSyntax namedA
 
                 if (parameter.IsParams && parameter.Type.IsSZArray() && i + 1 == parameterCount)
                 {
-                    reorderedArgument = GetParamArrayArgument(parameter, constructorArgsArray, argumentsCount, argsConsumedCount, this.Conversions);
+                    reorderedArgument = GetParamArrayArgument(parameter, constructorArgsArray, constructorArgumentNamesOpt, argumentsCount, argsConsumedCount, this.Conversions);
                     sourceIndices = sourceIndices ?? CreateSourceIndicesArray(i, parameterCount);
```

Comment:
>sourceIndices = sourceIndices ?? CreateSourceIndicesArray(i, parameterCount); [](start = 20, length = 77)

Should ```sourceIndices``` be adjusted differently for a named case?

Reply:
I'm looking into this. Thanks